api/v1alpha1/azurevalidator_types.go

```diff
@@ -55,6 +58,31 @@ type RBACRule struct {
 	PrincipalID string `json:"principalId" yaml:"principalId"`
 }
 
+// Conveys that one or more images in a community gallery exist.
```

Comment:
Please use a revive-friendly format, e.g.,
```
// AmiRule ensures that an EC2 AMI exists in a particular region.
```
Since you'll need to revive-ify this entire repo soon anyway.